Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:14.16.1-alpine
 
 LABEL maintainer="madnificent@gmail.com"
 
-RUN apk update && apk upgrade && apk add --no-cache bash git openssh
+RUN apk update && apk upgrade && apk add --no-cache bash git jq
```

Comment:
removed ssh since I don't really see a usecase for it, but can add again if necessary?